src/article.ts

```diff
@@ -235,7 +281,7 @@ const fromCapi = (docParser: DocParser) => (content: Content): Article => {
             } else if (isFeature(content)) {
                 return { layout: Layout.Feature, ...articleFieldsWithBody(docParser, content) };
 
-            } else if (isReview(content)) {
+            } else if (isReview(content) && fields?.starRating) {
```

Comment:
As this is more or less what the `isReview` function is doing anyway, and I believe this is the only place `isReview` is used, perhaps we can drop it entirely and just do this:

```suggestion
            } else if ('starRating' in fields) {
```

Reply:
isReview does have a check to see if the rating is `0 | 1 |2 | 3 | 4 |5` which if it isn't, falling back to standard articles might be what we want